.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Tests
+on: [ push, pull_request ]
+jobs:
+  test:
+    name: test (Ruby ${{ matrix.ruby }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ '2.5', '2.6', '2.7' ]
```

Comment:
Regressed here, can we add in Ruby 3?

Reply:
No, the gemspec only allows Ruby 2. 

https://github.com/xneelo/persistent-cache/blob/0686104b9110faee96f6ab43e9bc4a478de7cec3/persistent-cache.gemspec#L19

Changing this is out of scope for this PR.

Reply:
@JoshuaSmeda I've proposed this change in #10.